### PR TITLE
add anonymize member endpoint

### PR DESCRIPTION
**Pull request by wubini**
Adds doc for anonymize member endpoint

Unfortunately this also has some auto-formatting changes to remove whitespace

### CODE REVIEW

File: source/includes/_authenticated_api_members.md [hunk 1]
```diff
@@ -100,9 +100,33 @@ Permanently deletes a member with specified id and all associated records includ
...
+{
+  "member": {
+    "deleted": true,
+    "id": 123,
+    "email": "foo@bar.com"
+  }
+}
+```
+
+Anonymizes the member with specified id. This permanently deletes the member and all associated records including petitions, signatures, events. However, petition signatures from the anonymized member are preserved, anonymized, on petition data exports, signature counts, and petition letters. The method of anonymization is configured in organisation name privacy settings.
```
Comment (woodhull) [inline source/includes/_authenticated_api_members.md]: Petitions and events are not deleted. They are reassigned.

File: source/includes/_authenticated_api_members.md [hunk 2]
```diff
@@ -100,9 +100,33 @@ Permanently deletes a member with specified id and all associated records includ
...
+
+`POST /api/v1/members/123/anonymize`
+
+Deletes are synchronous and may take several seconds depending on how many resources are owned by the member.
```
Comment (woodhull) [inline source/includes/_authenticated_api_members.md]: Anonymization is synchronous...

**Review by woodhull — APPROVED** [review]
Just some small tweaks to make things more clear.

File: source/includes/_authenticated_api_members.md (hunk 1 above)
Comment (wubini) [reply source/includes/_authenticated_api_members.md]: `deletes the member and all associated signatures`?

**Pull request merged**